sdcm/sct_config.py

```diff
@@ -202,6 +203,16 @@ class SCTConfiguration(dict):
                      Automatically lookup AMIs and repo links for formal versions.
                      WARNING: can't be used together with 'scylla_repo' or 'ami_id_db_scylla'"""),
 
+        dict(name="user_data_format_version", env="SCT_USER_DATA_FORMAT_VERSION",
```

Comment:
Why do we need this additional params? Can't we just take the version from image tag?

Reply:
the idea is that we can overwrite it if needed, and that it would be available to the provision code and the *_cluster.py code in the same manner